skimage/measure/_regionprops.py

```diff
@@ -358,22 +358,93 @@ def __eq__(self, other):
         return True
 
 
+def _props_to_dict(regions, properties=('label', 'bbox'), separator='-', always_include_label=True):
+    """Convert image regions properties into a dictionary
+
+    Parameters
+    ----------
+    regions : (N,) list
+        List of _RegionProperties objects as returned by :func:`regionprops`
+    properties : tuple, optional
+        properties which the output need to have
+    separator : str, optional
+        Each element of non-scalar properties and is not listed in object_columns will
+        be put into its own column, with the index of that element separated from
+        the name by this separator.
+    always_include_label : bool, optional
+        allows autmatic adding of label to the output
+
+
+    Output
+    ------
+    A dictionary with properties in accordance to the wanted properties
+
+    Examples
+    --------
+    >>> from skimage import data, util, measure
+    >>> image = data.coins() > 110
+    >>> label_image = measure.label(image, connectivity=image.ndim)
+    >>> propslist = regionprops(label_image, image)
+    >>> props = to_dict(propslist, properties=['label', 'inertia_tensor', 'inertia_tensor_eigvals'])
+    >>> props
+    <output here>
```

Comment:
to clarify, you should run the code to find out the exact output, and paste that here. Same below.